src/store/models/network.ts

```diff
@@ -943,6 +978,79 @@ const networkModel: NetworkModel = {
 
     actions.setAutoMineMode({ id, mode });
   }),
+  renameNode: thunk(
+    async (actions, { node, newName }, { getState, getStoreActions, injections }) => {
+      if (!newName) throw new Error(l('renameErr', { newName }));
+
+      if (node.status === Status.Started) {
+        await actions.stop(node.networkId);
+      }
```

Comment:
This stops the network but it should also start it back up after performing the rename.